main.go

```diff
@@ -21,13 +21,15 @@ import (
 	"os"
 
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
```

Comment:
Controller-runtime has kubernetes client, why to use client from client-go?

Reply:
`client-go` simplifies deployment of StatefulSet, ReplicaSet, etc.